### DISCUSSION TITLE
win 11 printer issues

### DISCUSSION THREAD

**Discussion by MikeCook859** [robhagemans/pcbasic#222] [Q&A] [answered]
I was using PC-BASIC on windows 10 for several months.  I love it.   For some reason, I upgraded to win11.  Don't ask me why.  Now, when I try to print a program listing, I get an error message, I think from Windows, along the lines of, "this file does not have an associated app for printing."  I tried reinstalling PC-BASIC and using CCleaner to repair registry items, to no avail.  This is screwing up my project.  Does anyone have useful insight or advice?  By all means, let's hear it!  What am I going to do?  Thanks!

**Comment by robhagemans** [comment, marked as answer]
Can't say I'm a Windows expert, but printing in PC BASIC works through the
file associations for text files. Is there still a Notepad in Windows 11
and if so do you have it associated to text files? There should be a
Windows setting for associations somewhere, though I don't know where.

**Reply by MikeCook859** [reply]
Oh!  You know what, I switched that association from Notepad to Notepad++, do you think that broke it?  I can switch back.  BTW, thanks for the response.  That was super fast!

**Reply by MikeCook859** [reply]
Rob, I switched that default back to Notepad and that fixed it.  Thanks for the inspirado, it would have taken me a while to figure it out.  Thanks for PC-BASIC, too, I love it!

**Comment by robhagemans** [comment]
I think Notepad++ unfortunately doesn't implement the print-to association
which probably explains. Good to hear you enjoy using it!
